saas/axops/src/applatix.io/axops/configuration/configuration.go

```diff
@@ -23,10 +28,21 @@ type ConfigurationContext struct {
 	Key  string
 }
 
-const (
-	ConfigurationStrRegex = "^%%config\\.([^ ]*)\\.([-0-9A-Za-z_]+)\\.([-0-9A-Za-z_]+)%%$"
+type SecretResult struct {
+	SecretData     map[string]string `json:"data"`
+	SecretMetadata map[string]string `json:"metadata"`
+}
+
+var (
+	ConfigurationStrRegex = regexp.MustCompile("^%%config\\.([^ ]*)\\.([-0-9A-Za-z_]+)\\.([-0-9A-Za-z_]+)%%$")
```

Comment:
underscore is not supported by kubernetes secrets for keys. https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually